other/docker/windows/Dockerfile

```diff
@@ -1,10 +1,13 @@
-FROM debian:bullseye-slim
+FROM debian:bookworm-slim
+
+# When editing, make sure to update /other/windows_build_script_toxcore.sh and
+# INSTALL.md to match.
 
 # Build-time environment variables
-ARG VERSION_MSGPACK=4.0.0 \
-    VERSION_SODIUM=1.0.19 \
-    VERSION_OPUS=1.3.1 \
-    VERSION_VPX=1.11.0 \
+ARG VERSION_SODIUM=1.0.19 \
```

Comment:
While you're at it, maybe sort these alphabetically?

Reply:
The order is more of: toxcore deps, then toxav deps.

Reply:
Done